ppl/src/main/java/org/opensearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -106,6 +110,21 @@ public UnresolvedPlan visitSearchFilterFrom(SearchFilterFromContext ctx) {
         visit(ctx.fromClause()));
   }
 
+  /**
+   * Describe command.
+   */
+  @Override
+  public UnresolvedPlan visitDescribeCommand(DescribeCommandContext ctx) {
+    // Gets the mapping table (_ODFE_SYS_TABLE_MAPPINGS) of the indices by name
+    final String tableName = ctx.tableSource()
+        .stream()
+        .map(this::internalVisitExpression)
+        .map(UnresolvedExpression::toString)
+        .collect(Collectors.joining(","));
```

Comment:
Relation already has getTable() method which will compose the table name by using ','.

Reply:
Thank you! good to know